keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1363,8 +1363,9 @@ time to an Authorizer object supporting receiver-style function overloads:
 |-------------|-------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
 | path        | Authorizer.(path string) -> PathCheck                                   | Defines a check for an non-resource request path (e.g. /healthz)                                |
 | check       | PathCheck.(httpRequestVerb string) -> Decision                          | Checks if the user is authorized for the HTTP request verb on the path                          |
-| resource    | Authorizer.(kind string, group string, version string) -> ResourceCheck | Defines a check for API resources                                                               |
-| subresource | ResourceCheck.(subresource string) -> ResourceCheck                     | Specifies thath the check is for a subresource                                                  |
+| resource    | Authorizer.(resource string) -> ResourceCheck                           | Defines a check for an API resource                                                             |
+| group       | ResourceCheck.(group string) -> ResourceCheck                           | Specifies the group for the API resource check                                                  |
```

Comment:
I think both `group` and `resource` should be required to be called to submit a resourcecheck authz call - otherwise people will unintentionally be doing authz checks against the core API group when they didn't intend to.

an explicit example below to authorize something in the core API group (`...group('')`) would also probably be good

Reply:
Should this just be a single call that takes the fully qualified resource name instead? E.g. `roles.rbac.authorization.k8s.io`?

Reply:
Making the fields match what is in RBAC, SubjectAccessReview, and authorizer calls seems better than concatenating to me

Reply:
Feedback applied.